test/api/v3/integration/chat/POST-chat.test.js

```diff
@@ -94,10 +94,10 @@ describe('POST /chat', () => {
       error: 'BadRequest',
       message: 'Your message contained inapropriate language, and your chat privileges have been revoked.',
     });
+
     expect(user.flags.chatRevoked).to.be.true;
 
-    user.flags.chatRevoked = false;
-    await user.save();
+    await user.update({'flags.chatRevoked': false});
```

Comment:
consider that this only sends a command to mongo, so the database will have `flags.chatRevoked` set to false but the local representation of the user will not be updated, so `user.flags.chatRevoked` won't be `false` but still the old value